pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -114,7 +115,8 @@ private TransactionWrapper(int maxConnections, String dbUrl, DB db) {
      * Should only be called during testing.
      */
     public static synchronized void reset() {
-        log.warn("reset() should only be called in the context of testing");
+        log.warn("reset() should only be called in the context of testing.  Call stack follows");
+        log.warn(Arrays.toString(Thread.currentThread().getStackTrace()).replace(',', '\n'));
```

Comment:
I added a stack trace here when trying to reproduce some circleCI issues locally.  Thoughts on keeping it around?

Reply:
I'm going to ditch this and revert to the original, simpler warn.